docs/reference/cluster/nodes-reload-secure-settings.asciidoc

```diff
@@ -4,31 +4,47 @@
 <titleabbrev>Nodes reload secure settings</titleabbrev>
 ++++
 
-
 The cluster nodes reload secure settings API is used to re-load the keystore on each node.
 
-[source,console]
---------------------------------------------------
-POST _nodes/reload_secure_settings
-POST _nodes/nodeId1,nodeId2/reload_secure_settings
---------------------------------------------------
-// TEST[setup:node]
-// TEST[s/nodeId1,nodeId2/*/]
+[[cluster-nodes-reload-secure-settings-api-request]]
+==== {api-request-title}
+`POST _nodes/reload_secure_settings` +
+
+`POST _nodes/nodeId1,nodeId2/reload_secure_settings` +
+
+`POST _nodes/_local/reload_secure_settings`
```

Comment:
I think this can be omitted since it's just another example of a <nodes> value and it's mentioned in the link from the path parameters section.